app/components/helpers/HelpButton.tsx

```diff
@@ -66,7 +67,25 @@ const HelpButton: React.FunctionComponent<Props> = ({
               </li>
             </ul>
           ) : (
-            ''
```

Comment:
Perhaps better than duplicating the help bubble content for reporters (which could  facilitate accidental template divergence) would be to extract constants, `option1` and `option2`:
```
const option1 = (
  <>
    span (based on user)
    anchor link (based on user)
  </>
);

...

<ul>
  <li>{option1}</li>
  <li>OR</li>
  <li>{option2}</li>
</ul>
```

Reply:
Yeah if we're sticking to only 2 options each we could do this.